### DISCUSSION TITLE
[Feature] Add option to do something with selected items from a search query

### DISCUSSION THREAD

**Discussion by JVT038** [immich-app/immich#2114] [Feature Request]
### The feature

When I select results from a search query, I can't do anything with them. 
I can't add them to an album, download them, share them, or delete them.
See web interface example for what I'd like in the search bar on web:
![image](https://user-images.githubusercontent.com/47184046/228578397-078b2d99-1191-4ffc-b933-ee2f076f5e3a.png)

On mobile, I can't select any items at all.
So I'd like to be able to select items on the search page and do something with them, just like I can select items on the main timeline and do something.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bin16** [comment]
+1 the current web search:

<img width="1440" alt="图片" src="https://user-images.githubusercontent.com/11701531/229390151-22052db9-7471-41e0-9d71-b9a46ffef266.png">

could select multiple photos, but no actions available. would be nice to have buttons to add them to albums
